### PR TITLE
chore: Add widget id header RELEASE

### PR DESCRIPTION
**Pull request by dorsha**

### CODE REVIEW

**Review by aviadl — APPROVED** [review]

**Comment by meirwah** [comment]
> Deployment failed with the following error:
> 
> ```
> The Preview Deployment Suffix domain "preview.descope.org" doesn't point to Vercel nameservers. Learn more: https://vercel.link/3a1
> ```

@newman-dani ?

**Comment by dorsha** [comment]
Looks like an issue with vercel.

File: packages/access-key-management-widget/src/lib/widget/api/sdk/index.ts [hunk 1]
```diff
@@ -6,8 +6,16 @@ export const createSdk = (
...
   tenant: string,
   mock: boolean,
+  widgetId: string,
 ) => {
-  const webSdk = createWebSdk({ ...config, persistTokens: true });
+  const webSdk = createWebSdk({
+    ...config,
+    persistTokens: true,
+    baseHeaders: {
+      'x-descope-widget-type': 'access-key-management-widget',
```
Comment (nirgur) [inline packages/access-key-management-widget/src/lib/widget/api/sdk/index.ts:15]: Do we need the widget version as well?

**Review by nirgur — APPROVED** [review]

File: packages/access-key-management-widget/src/lib/widget/api/sdk/index.ts (hunk 1 above)
Comment (dorsha) [reply packages/access-key-management-widget/src/lib/widget/api/sdk/index.ts:15]: no, it will be propagat from outside

Comment (dorsha) [reply packages/access-key-management-widget/src/lib/widget/api/sdk/index.ts:15]: image

Comment (nirgur) [reply packages/access-key-management-widget/src/lib/widget/api/sdk/index.ts:15]: This is the web-sdk version, not the widget version

**Review by aviadl — APPROVED** [review]

**Review by aviadl — DISMISSED** [review]

**Comment by dorsha** [comment]
Need to add user profile here as well: https://github.com/descope/descope-js/pull/524

**Review by nirgur — APPROVED** [review]

**Review by nirgur — DISMISSED** [review]
There is an error when calling `/me`

image

**Comment by dorsha** [comment]
Yes, because need to add that header to the backend.

**Review by aviadl — APPROVED** [review]

**Pull request merged**